Assets/Scripts/Player/Weapons/Kinetic/Pistol.cs

```diff
@@ -0,0 +1,5 @@
+namespace OperationBlackwell.Player {
+	public class Pistol: Weapon {
```

Comment:
I think the other files have a space between `Pistol` and `:`, Check it and change this if they do.